polygeist/lib/Conversion/PolygeistToLLVM/PolygeistToLLVM.cpp

```diff
@@ -302,6 +338,11 @@ struct Memref2PointerOpLowering
   LogicalResult
   matchAndRewrite(Memref2PointerOp op, OpAdaptor transformed,
                   ConversionPatternRewriter &rewriter) const override {
+    if (LogicalResult verifResult = verifyPtrMemrefConversion(
```

Comment:
```suggestion
    if (LogicalResult verifyResult = verifyPtrMemrefConversion(
```
and other `verifResult`s.